Project.toml

```
@@ -14,7 +14,7 @@ p7zip_jll = "3f19e933-33d8-53b3-aaab-bd5110c3b7a0"
 [compat]
 BinaryProvider = "0.5"
 ExpectationStubs = "0.2"
-HTTP = "0.8.5, 0.9"
+HTTP = "0.8.5, 0.9, 1"
```

this uses a new keyword argument to download so breaks compat
```suggestion
HTTP = "1"
```

---

I think `HTTP.download` in old version drop unknow keywords. I do try this patch with HTTP 0.9 and still pass the tests, though maybe not a good way  to do.

---

hmm, ok.
I am still happy enough dropping support unless someone specifically requests it